src/OpenTelemetry.Instrumentation.Http/README.md

```diff
@@ -62,14 +65,51 @@ public class Program
 }
 ```
 
+#### Metrics
+
+The following example demonstrates adding HTTP client instrumentation with the
+extension method `.AddHttpClientInstrumentation()` on `MeterProviderBuilder` to
+a console application. This example also sets up the OpenTelemetry Console
+Exporter, which requires adding the package
+[`OpenTelemetry.Exporter.Console`](../OpenTelemetry.Exporter.Console/README.md)
+to the application.
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddHttpClientInstrumentation()
+            .AddConsoleExporter()
+            .Build();
+    }
+}
+```
+
 For an ASP.NET Core application, adding instrumentation is typically done in the
 `ConfigureServices` of your `Startup` class. Refer to documentation for
 [OpenTelemetry.Instrumentation.AspNetCore](../OpenTelemetry.Instrumentation.AspNetCore/README.md).
+In addition, check how to enable `OpenTelemetry.Instrumentation.Http` for Trace
```

Comment:
I'd suggest to remove the previous 2 sentences. Its not really needed here, since we are linking to a proper example app.